src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -405,8 +459,11 @@ public void testExistingIncorrectChildrenInConfigFile()
             throws Exception {
         exit.expectSystemExitWithStatus(-2);
         exit.checkAssertionAfterwards(() -> {
+            final LocalizedMessage errorCounterMessage = new LocalizedMessage(0,
+                    Definitions.CHECKSTYLE_BUNDLE, ERROR_COUNTER_MESSAGE,
+                    new String[] {String.valueOf(1)}, null, getClass(), null);
```

Comment:
please move this variable to class field and use it in all cases. You have already 7 of them